### PR TITLE
Fluid API 

### PR DESCRIPTION
**Pull request by geNAZt**

### CODE REVIEW

File: gomint-api/src/main/java/io/gomint/i18n/LocaleManager.java [hunk 1]
```diff
@@ -271,7 +271,7 @@ public String translate( String translationKey, Object... args ) {
      *
      * @param loader which is used to load specific locale resources
      */
-    public void registerLoader( ResourceLoader loader ) {
+    public void registerLoader( ResourceLoader<?> loader ) {
```
Comment (fyradir) [inline gomint-api/src/main/java/io/gomint/i18n/LocaleManager.java:274]: Shouldn't this be "fluid" too?

File: gomint-api/src/main/java/io/gomint/i18n/localization/ResourceManager.java [hunk 2]
```diff
@@ -59,7 +59,7 @@ public ResourceManager( ClassLoader classLoader ) {
      *
      * @param loader New loader which can be used to load Resources
      */
-    public synchronized void registerLoader( ResourceLoader loader ) {
+    public synchronized void registerLoader( ResourceLoader<?> loader ) {
```
Comment (Soupertonic) [inline gomint-api/src/main/java/io/gomint/i18n/localization/ResourceManager.java]: This function should return the class its declared in.

File: gomint-api/src/main/java/io/gomint/i18n/LocaleManager.java (hunk 1 above)
Comment (Soupertonic) [inline gomint-api/src/main/java/io/gomint/i18n/LocaleManager.java]: This function should return the class its declared in.

File: gomint-server/src/main/java/io/gomint/server/entity/ai/AIFollowEntity.java [hunk 3]
```diff
@@ -45,7 +45,7 @@ public AIFollowEntity( AIStateMachine machine, WorldAdapter world, PathfindingEn
      *
      * @param entity the new entity to follow
      */
-    public void setFollowEntity( Entity entity ) {
+    public void setFollowEntity( Entity<?> entity ) {
```
Comment (Soupertonic) [inline gomint-server/src/main/java/io/gomint/server/entity/ai/AIFollowEntity.java]: Change function name to `followEntity` and return the class its declared in.

**Review by Soupertonic — CHANGES_REQUESTED** [review]

**Review by Soupertonic — APPROVED** [review]

**Pull request merged**